conbench/job.py

```diff
@@ -123,10 +123,15 @@ class CacheDict(TypedDict):
 SHUTDOWN = False
 _STARTED = False
 
+BMRT_CACHE_SIZE = 0.2 * 10**6
+if Config.TESTING:
+    # quicker update in testing
+    BMRT_CACHE_SIZE = 0.2 * 10**6
```

Comment:
Ha. :D Next time. In this case, I actually wanted to have the big cache size for local dev.